### PR TITLE
Update dependencies

### LINKED ISSUE #4: Rollup crash
Both rollup commands crash with
```
Cannot read property 'Symbol(Symbol.iterator)' of undefined
TypeError: Cannot read property 'Symbol(Symbol.iterator)' of undefined
    at enhance (/home/mihail/Documents/Development/svelte-ssr-bundle/node_modules/rollup/src/ast/enhance.js:12:25)
    at Module.analyse (/home/mihail/Documents/Development/svelte-ssr-bundle/node_modules/rollup/src/Module.js:214:2)
    at new Module (/home/mihail/Documents/Development/svelte-ssr-bundle/node_modules/rollup/src/Module.js:87:8)
    at /home/mihail/Documents/Development/svelte-ssr-bundle/node_modules/rollup/src/Bundle.js:272:20
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:611:11)
    at startup (bootstrap_node.js:158:16)
    at bootstrap_node.js:598:3
```
updating to latest `rollup@0.50.0, rollup-plugin-svelte@3.1.0` fixes it.

### PR DESCRIPTION
**Pull request by qm3ster**
closes #4

### CODE REVIEW

**Comment by qm3ster** [comment]
Henlo @Rich-Harris

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "devDependencies": {
-    "rollup": "^0.37.0",
-    "rollup-plugin-svelte": "^1.3.0"
+    "rollup": "^0.50.0",
+    "rollup-plugin-svelte": "^3.1.0"
   },
```
Comment (iegik) [inline package.json:5]: Also need add svelte as dependency!?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "devDependencies": {
-    "rollup": "^0.37.0",
-    "rollup-plugin-svelte": "^1.3.0"
+    "rollup": "^0.50.0",
```
Comment (iegik) [inline package.json:3]: ^1.31.1 works

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+    "rollup-plugin-svelte": "^3.1.0"
```
Comment (iegik) [inline package.json:4]: ^5.1.1 works

**Pull request closed without merging**